woostarter/style.css

```diff
@@ -21,3 +21,18 @@ Tags: e-commerce, one-column, two-columns, three-columns, four-columns, wide-blo
     transition-duration: 0.15s;
     transition-timing-function: cubic-bezier(0.25, 0.46, 0.45, 0.94);
 }
+
+/* Product categories. */
+.wp-block-post-terms:not(.taxonomy-product_cat) a {
```

Comment:
we should post links to Woo issues to remind ourselves to remove these in the future